sklearn/metrics/classification.py

```diff
@@ -934,7 +925,7 @@ def precision_recall_fscore_support(y_true, y_pred, beta=1.0, labels=None,
         labels = np.hstack([labels, np.setdiff1d(present_labels, labels,
                                                  assume_unique=True)])
 
-    ### Calculate tp_sum, pred_sum, true_sum ###
+    ###Calculate tp_sum, pred_sum, true_sum ###
```

Comment:
IIRC this already is a revert :P (this was how it was earlier and to make it PEP8 compliant I changed it to `###<space>` but later we decided to keep such changes out of this PR... do you suggest to make it `###<space>` again so that it is PEP8 compliant? :)